R/extract.R

```diff
@@ -7,6 +7,33 @@
 setGeneric("extract", function(model, ...) standardGeneric("extract"), 
     package = "texreg")
 
+# default extract method prompts users to install the broom package
+extract.default <- function(model, ...) {
```

Comment:
I am not familiar with `default` class names. Is this what is executed if the class name is not recognized? Or does `broom` really call its objects `default`?